config/backup.php

```diff
@@ -40,6 +40,13 @@
                  * Determines if it should avoid unreadable folders.
                  */
                 'ignore_unreadable_directories' => false,
+
+                /*
+                 * This path is used to make directories in resulting zip-file relative
+                 * Set to false to include complete absolute path
+                 * Example: base_path()
+                 */
+                'relative_path' => false,
```

Comment:
Most of our config options start with a verb. Rename this to `use_relative_path` for consistency/

Reply:
Actually, it works by providing a path. So if I set 'relative_path' to '/Users/marnick/' It will remove that part from the path in the zip-file. Not sure if that is clear enough explained though.

I might change the option to a boolean, and just remove the base_path() from all included files if set to true. But for my use case I needed a more flexible option (dealing with tenancy).